ArticleTemplates/assets/js/modules/immersive.js

```diff
@@ -103,7 +103,7 @@ function onImmersiveScroll() {
 }
 
 function getImageHeight() {
```

Comment:
Only [android](https://github.com/guardian/mobile-apps-article-templates/blob/master/ArticleTemplates/assets/js/modules/immersive.js#L146) uses this function (for a reason I don't quite understand)

This styling is applied directly to the element (the header background image) - we need this change here to change the styling on android.